docs/pbc/all/tax-management/202307.0/vertex/install-vertex.md

```diff
@@ -0,0 +1,422 @@
+
+class OrderCustomerWithVertexCodeExpanderPlugin extends AbstractPlugin implements OrderTaxAppExpanderPluginInterface
+{
+    /**
+     * @param \Generated\Shared\Transfer\OrderTransfer $orderTransfer
+     *
+     * @return \Generated\Shared\Transfer\OrderTransfer
+     */
+    public function expand(OrderTransfer $orderTransfer): OrderTransfer
+    {
+        $orderTransfer->getTaxMetadata()->setCustomer(['customerCode' => 'vertex-customer-code']);
+
+        return $orderTransfer;
+    }
+}
+```
+
+The following code sample shows how to introduce the following expander plugin:
+
+`Pyz/Zed/{YourDesiredModule}/Communication/Plugin/Quote/CalculableObjectCustomerWithVertexCodeExpanderPlugin.php`
+
+
+```php
+<?php
+
+namespace Spryker\Zed\{YourDesiredModule}\Communication\Plugin\Quote;
+
+use Generated\Shared\Transfer\CalculableObjectTransfer;
+use Spryker\Zed\Kernel\Communication\AbstractPlugin;
+use Spryker\Zed\TaxAppExtension\Dependency\Plugin\CalculableObjectTaxAppExpanderPluginInterface;
+
+class CalculableObjectCustomerWithVertexCodeExpanderPlugin extends AbstractPlugin implements CalculableObjectTaxAppExpanderPluginInterface
+{
+    /**
+     * @param \Generated\Shared\Transfer\CalculableObjectTransfer $quoteTransfer
+     *
+     * @return \Generated\Shared\Transfer\CalculableObjectTransfer
+     */
+    public function expand(CalculableObjectTransfer $quoteTransfer): CalculableObjectTransfer
+    {
+        $quoteTransfer->getTaxMetadata()->setCustomer(['customerCode' => 'vertex-customer-code']);
+
+        return $quoteTransfer;
+    }
+}
+```
+
+#### Product Class Code Expanders
+
+For order items:
+
+`Pyz/Zed/{YourDesiredModule}/Communication/Plugin/Order/OrderItemVertexProductClassCodeExpanderPlugin.php`
+
+and for quote items:
+
+`Pyz/Zed/{YourDesiredModule}/Communication/Plugin/Quote/CalculableObjectItemWithVertexProductClassCodeExpanderPlugin.php`
+
+The contents of both plugins are similar:
+
+```php
+// ...
+class ItemWithVertexClassCodeExpanderPlugin extends AbstractPlugin implements CalculableObjectTaxAppExpanderPluginInterface
+{
+    /**
+     * @param \Generated\Shared\Transfer\CalculableObjectTransfer $calculableObjectTransfer
+     *
+     * @return \Generated\Shared\Transfer\CalculableObjectTransfer
+     */
+    public function expand(CalculableObjectTransfer $calculableObjectTransfer): CalculableObjectTransfer
+    {
+        foreach ($calculableObjectTransfer->getItems() as $itemTransfer) {
+            $itemTransfer->getTaxMetadata()->setProduct(['productClass' => 'vertex-product-class-code']);
+        }
+
+        return $calculableObjectTransfer;
+    }
+}
+```
+
+{% info_block infoBox "Use same Product Class Code" %}
+
+The same Product Class Code extension must be used for all the product options and other order expenses because, in Vertex's perspective, all of them are separate items for tax calculation. To find them a proper place, you can refer to the transfers' definition, which was outlined above.
```

Comment:
Use respective headings to refer to sections/documents - replace "which was outlined above" with "which is outlined in the [section title](#sectio title).